client/src/components/Map/Map.jsx

```diff
@@ -68,6 +69,10 @@ class Map extends Component {
 
     // When a new place is selected the map will be forced to update
     this.autocomplete.addListener("place_changed", this.handleMapChange);
+
+    document
+      .getElementById("searchButton")
+      .addEventListener("click", this.handleMapChange);
```

Comment:
due to the nature of the DOM it is best to create a ref to latch on to DOM elements rather than using query selection. With the virtual DOM you don't know if this element will be mounted or changed from the original so this might cause a memory leak or other errors. Take a look at this if you are not sure how to use refs in react https://reactjs.org/docs/refs-and-the-dom.html

Reply:
I wasn't able to resolve this, I will have to pick back up with it tomorrow.

Reply:
Updated now